README.md

```diff
@@ -91,6 +91,12 @@ app.listen(80, function () {
 
 ### Configuring CORS w/ Dynamic Origin
 
+Provide an `origin` function, which will be passed a string `origin` containing the origin domain, and a `callback` expecting `(err, ok)`.
+
+If you callback with `(null, true)`, the incoming origin will be echoed back in the access-control-allow-origin header, permitting browser acces to the resource. Callback with `(null, false)` and it won't, and the browser will abort the cross-origin request.
+
+Calling back with an error should be avoided, unless you really mean to block all access to your server from the passed origin - it will halt the request and return an error to the client, which ends up rejecting same-origin requests and requests from non-browser clients like curl or Postman if you're following a typical 'whitelist' approach.
```

Comment:
This should probably clarify what the `err` is used for: to indicate if there was an error checking if the original should or should not be allowed. The error is handed down the Express.js error pipeline and will perform whatever that pipeline is configured to do in the app, though keeping a note that no CORS response headers will be added by this middleware is still valid 👍